gef-ui/server/controllers/application-abandon/index.js

```diff
@@ -39,7 +39,7 @@ const abandonApplication = async (req, res, next) => {
   try {
     const application = await Application.findById(dealId, user, userToken);
```

Comment:
Pass object as argument?

Reply:
I'm going to leave it since I didn't edit the signature of this function, just to limit the scope of this ticket :)